### ISSUE TITLE
Browsing issue between collections

### ISSUE THREAD

**Issue by DGMayor** [MediaBrowser/Emby#203]
While browsing through my movies collection (my biggest) to updated watched indicators, I got to the end (page 29) and it was listed as 561-564 of 564.

I then went back to the main page, and if I go to any other collection, nothing displays, they list as 561-15 of 15, 561-112 of 112, etc.

I have to go back into the large collection, go back to page 1 and then exit and return to any of the smaller collections to see anything.

tl:dr version:

paginated view of collection isn't resetting the page between collections, if you leave collection 1 on page 29, you enter the next on page 29 even if there is only 1 page.
